rotkehlchen/api/v1/schemas.py

```diff
@@ -734,49 +739,155 @@ def generate_fields_post_validation(self, data: dict[str, Any]) -> dict[str, Any
         return {'group_by_event_ids': data['group_by_event_ids']}
 
 
-class EvmEventSchema(Schema):
+class CreateHistoryEventSchema(Schema):
     """Schema used when adding a new event in the EVM transactions view"""
-    tx_hash = EVMTransactionHashField(required=True)
-    sequence_index = fields.Integer(required=True)
-    # Timestamp coming in from the API is in seconds, in contrast to what we save in the struct
-    timestamp = TimestampField(ts_multiplier=1000, required=True)
-    location = LocationField(required=True)
-    event_type = SerializableEnumField(enum_class=HistoryEventType, required=True)
-    asset = AssetField(required=True, expected_type=Asset, form_with_incomplete_data=True)
-    balance = fields.Nested(BalanceSchema, required=True)
-    location_label = fields.String(required=False)
-    notes = fields.String(required=False)
-    event_subtype = SerializableEnumField(
-        enum_class=HistoryEventSubType,
-        required=False,
-        load_default=HistoryEventSubType.NONE,
-        allow_none=True,
-    )
-    counterparty = fields.String(load_default=None)
-    product = SerializableEnumField(enum_class=EvmProduct, load_default=None)
-    address = EvmAddressField(load_default=None)
+    include_identifier: bool = False
+    entry_type = SerializableEnumField(enum_class=HistoryBaseEntryType, required=True)
+
+    class BaseSchema(Schema):
+        # Timestamp coming in from the API is in seconds, in contrast to what we save in the struct
+        timestamp = TimestampField(ts_multiplier=1000, required=True)
+        balance = fields.Nested(BalanceSchema, required=True)
+        identifier = fields.Integer(required=True)
+
+    class BaseEventSchema(BaseSchema):
+        event_type = SerializableEnumField(enum_class=HistoryEventType, required=True)
+        event_subtype = SerializableEnumField(
+            enum_class=HistoryEventSubType,
+            required=False,
+            load_default=HistoryEventSubType.NONE,
+            allow_none=True,
```

Comment:
As mentioned below:

1.  Remove this
2. Communicate the change in the api to the fronted
3. Edit the docs


```suggestion
```

Reply:
@rotki/frontend event_subtype does not accept null anymore, it accepts the string "none"